geopandas/tests/test_plotting.py

```diff
@@ -1226,6 +1226,11 @@ def test_schemes(self):
         for scheme in self.classifiers:
             self.df.plot(column="pop_est", scheme=scheme, legend=True)
 
+    def test_greedy_mapclassifier(self):
+        # test if greedy classifier passes
+        pytest.importorskip("libpysal")
```

Comment:
After installing from `requirements-dev.txt` I had issues with this package not being installed so added this here. But looking at other dependency files I'm wondering if it should be in the dev requirements after all?